### ISSUE TITLE
Cannot start nuxt:  Cannot find module 'nuxt3' 

### ISSUE THREAD

**Issue by Droutin** [nuxt/framework#4507]
### Environment

- Operating System: `Darwin`
- Node Version:     `v16.14.2`
- Nuxt Version:     `3.0.0-rc.1`
- Package Manager:  `npm@8.7.0`
- Builder:          `vite`
- User Config:      `runtimeConfig`, `modules`, `intlify`, `css`, `head`, `vite`
- Runtime Modules:  `@intlify/nuxt3@0.1.10`
- Build Modules:    `-`

### Reproduction

-

### Describe the bug

Commands `npx nuxi dev` and `npx nuxi build` throws error. Error can be seen in logs 

### Additional context

CONFIG
```ts
import { defineNuxtConfig } from "nuxt";

export default defineNuxtConfig({
  runtimeConfig: {
    DATABASE_URL: process.env.DATABASE_URL,
    CORAGEO_URL: process.env.CORAGEO_URL,
    CORAGEO_USER: process.env.CORAGEO_USER,
    CORAGEO_PASS: process.env.CORAGEO_PASS,
    CORAGEO_VERBOSE: process.env.CORAGEO_VERBOSE,
    ORGID: process.env.ORGID,
    UPLOAD_DIR: process.env.UPLOAD_DIR,
    API_URL: process.env.API_URL,
    EMAIL_VERIFICATION_EXPIRATION_DAYS: process.env.EMAIL_VERIFICATION_EXPIRATION_DAYS,
    PLATFORM: process.env.PLATFORM,
    SESSION_LIFESPAN_MINUTES: process.env.SESSION_LIFESPAN_MINUTES,
    RESET_PASS_EXPIRATION_DAYS: process.env.RESET_PASS_EXPIRATION_DAYS,
    public: {
      MAXSIZEMB: process.env.MAXSIZEMB,
      PAYMENT_URL: process.env.PAYMENT_URL,
      PAYMENT_KEY: process.env.PAYMENT_KEY,
    },
  },
  modules: ["@intlify/nuxt3"],
  intlify: {
    localeDir: "locales",
    vueI18n: {
      globalInjection: true,
      legacy: false,
      locale: "sk",
    },
  },
  css: ["@/assets/scss/app.scss"],
  head: {
    meta: [{ name: "viewport", content: "width=device-width, initial-scale=1" }],
  },
  vite: {
    css: {
      preprocessorOptions: {
        scss: {
          additionalData: "@import '@id-sk/frontend/idsk/all.scss';",
          quietDeps: true,
        },
      },
    },
  },
});

```
PACKAGE.JSON
```json
{
  "private": true,
  "scripts": {
    "dev": "nuxi dev",
    "build": "nuxi build",
    "preview": "nuxi preview",
    "start": "cross-env NUXT_HOST=0.0.0.0 node .output/server/index.mjs",
    "format": "prettier --config .prettierrc . --write"
  },
  "devDependencies": {
    "@intlify/nuxt3": "^0.1.10",
    "cross-env": "^7.0.3",
    "nuxt": "^3.0.0-rc.1",
    "prettier": "^2.6.2",
    "prisma": "^3.11.1"
  },
  "dependencies": {
    "@id-sk/frontend": "^2.9.0",
    "@prisma/client": "^3.11.1",
    "@vue-leaflet/vue-leaflet": "^0.6.1",
    "axios": "^0.26.1",
    "base64-img": "^1.0.4",
    "dayjs": "^1.11.0",
    "elmolis-corageo": "git+ssh://git@gitlab.elmolis.sk:Urbi/IntParkCoraGeo.git#devel",
    "hashids": "^2.2.10",
    "leaflet": "^1.7.1",
    "mustache": "^4.2.0",
    "sass": "^1.49.11",
    "sweetalert2": "^11.4.8"
  }
}

```

### Logs

```shell
ERROR  Cannot start nuxt:  Cannot find module 'nuxt3'                                                                                                                                  15:31:03
Require stack:
- /Users/enimo/Desktop/Projects/Elmolis/PortalPresov/index.js

  Require stack:
  - index.js
  at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
  at Function.resolve (node:internal/modules/cjs/helpers:108:19)
  at Function._resolve [as resolve] (node_modules/jiti/dist/jiti.js:1:192841)
  at resolveModule (node_modules/@nuxt/kit/dist/index.mjs:68:29)
  at importModule (node_modules/@nuxt/kit/dist/index.mjs:90:24)
  at loadNuxt (node_modules/@nuxt/kit/dist/index.mjs:798:43)
  at async load (node_modules/nuxi/dist/chunks/dev.mjs:6733:23)
  at async Object.invoke (node_modules/nuxi/dist/chunks/dev.mjs:6777:5)
  at async _main (node_modules/nuxi/dist/cli.mjs:46:20)
```

**Comment by danielroe** [comment]
You may need to remove your package lockfile and `node_modules` and reinstall. See https://github.com/nuxt/framework/discussions/4479.

**Comment by Droutin** [comment]
@danielroe i tried that several times. And command `npx nuxi upgrade --force`

**Comment by danielroe** [comment]
It may be one of your dependencies then.

Would you provide a reproduction? 🙏

**Comment by fiamma06** [comment]
Have the same issue. Problem was in my custom plugins. 
<img width="378" alt="image" src="https://user-images.githubusercontent.com/1270206/164476471-cbf62931-8ae7-4df3-bd24-1870243a0511.png">
Solved by changing nuxt3 to nuxt repository.

**Comment by djschilling** [comment]
I am having the same issue after updating to the release candidate.

I replaced all my `nuxt3` imports with `nuxt`, removed `package-lock.json` , `node_modules` and `.nuxt`.

These are the packages i am using:

```
    "devDependencies": {
        "@fortawesome/fontawesome-free": "^5.15.4",
        "@headlessui/vue": "^1.4.3",
        "@intlify/nuxt3": "^0.1.10",
        "@tailwindcss/aspect-ratio": "^0.4.0",
        "@tailwindcss/forms": "^0.4.0",
        "@tailwindcss/line-clamp": "^0.3.1",
        "@tailwindcss/typography": "^0.5.0",
        "@types/leaflet": "^1.7.8",
        "@types/lodash": "^4.14.178",
        "@types/node": "^17.0.10",
        "@typescript-eslint/eslint-plugin": "^5.10.0",
        "@typescript-eslint/parser": "^5.10.0",
        "autoprefixer": "^10.4.2",
        "churchtools-styleguide": "git+ssh://git@gitlab.ct-srv.de:2222/churchtools/churchtools-styleguide.git#finder-1.0",
        "date-fns": "^2.28.0",
        "eslint": "^8.7.0",
        "eslint-plugin-vue": "^8.3.0",
        "leaflet": "^1.7.1",
        "leaflet.markercluster": "^1.5.3",
        "nuxt": "^3.0.0-rc.1",
        "postcss": "^8.4.5",
        "postcss-import": "^14.0.2",
        "prettier": "^2.5.1",
        "sass": "^1.49.0",
        "sass-loader": "^10.2.1",
        "tailwindcss": "^3.0.15",
        "ts-node": "^10.4.0",
        "typescript": "^4.5.5",
        "@vitejs/plugin-vue": "^2.3.0",
        "vite": "^2.8.0",
        "vue-virtual-scroller": "^2.0.0-alpha.1",
        "vue": "^3.2.25"
    },
    "dependencies": {
        "@types/leaflet.markercluster": "^1.4.6",
        "axios": "0.25.0",
        "axios-cookiejar-support": "^2.0.3",
        "cropperjs": "^1.5.12",
        "csv-parser": "^3.0.0",
        "dotenv": "^14.2.0",
        "lodash-es": "^4.17.21",
        "mysql2": "^2.3.3",
        "node-cron": "^3.0.0",
        "reflect-metadata": "^0.1.13",
        "tippy.js": "^6.3.7",
        "tough-cookie": "4.0.0",
        "typeorm": "^0.2.41",
        "winston": "^3.4.0"
    },
```

I thought maybe `@intlify/nuxt3` would be the problem, but searching their source code i could not find an import of `nuxt3` their either: https://github.com/intlify/nuxt3/search?p=1&q=nuxt3

**Comment by Droutin** [comment]
I edited all files to match new docs and still have the same problem.

```
@nuxt/kit@3.0.0-rc.0 dev
node_modules/@nuxt/telemetry/node_modules/@nuxt/kit
  @nuxt/kit@"3.0.0-rc.0" from @nuxt/telemetry@2.1.2
  node_modules/@nuxt/telemetry
    @nuxt/telemetry@"^2.1.2" from nuxt@3.0.0-rc.1
    node_modules/nuxt
      dev nuxt@"^3.0.0-rc.1" from the root project

@nuxt/kit@3.0.0-rc.1 dev
node_modules/@nuxt/vite-builder/node_modules/@nuxt/kit
  @nuxt/kit@"^3.0.0-rc.1" from @nuxt/vite-builder@3.0.0-rc.1
  node_modules/@nuxt/vite-builder
    @nuxt/vite-builder@"^3.0.0-rc.1" from nuxt@3.0.0-rc.1
    node_modules/nuxt
      dev nuxt@"^3.0.0-rc.1" from the root project

@nuxt/kit@3.0.0-rc.1 dev
node_modules/nuxt/node_modules/@nuxt/kit
  @nuxt/kit@"^3.0.0-rc.1" from nuxt@3.0.0-rc.1
  node_modules/nuxt
    dev nuxt@"^3.0.0-rc.1" from the root project
```

```
@nuxt/kit@3.0.0-27277498.850ef69 dev
node_modules/@nuxt/kit
  @nuxt/kit@"npm:@nuxt/kit-edge@3.0.0-27277498.850ef69" from @intlify/nuxt3@0.1.10
  node_modules/@intlify/nuxt3
    dev @intlify/nuxt3@"^0.1.10" from the root project
```

**Comment by Droutin** [comment]
`loadNuxt (node_modules/@nuxt/kit/dist/index.mjs:798:43)`

why is in this file `const { loadNuxt: loadNuxt3 } = await importModule("nuxt3", resolveOpts);`

**Comment by Droutin** [comment]
> It may be one of your dependencies then.
> 
> Would you provide a reproduction? 🙏

Send me your email.

**Comment by danielroe** [comment]
> why is in this file const { loadNuxt: loadNuxt3 } = await importModule("nuxt3", resolveOpts);

That is an outdated version of `@nuxt/kit`, 5 months old. It looks like intlify/nuxt3 needs to use `@latest` tag for the edge release: https://github.com/intlify/nuxt3/blob/main/packages/nuxt3/package.json#L46

cc: @kazupon

**Comment by kazupon** [comment]
I've just released v0.2.0 which is supported for nuxt3 rc version.
https://github.com/intlify/nuxt3/releases/tag/v0.2.0
Please check it
